core/src/http_helpers.rs

```diff
@@ -136,31 +136,26 @@ pub fn read_header_values<'a>(
 	headers.get_all(header_name)
 }
 
-#[cfg(test)]
```

Comment:
Can you revert this? 

I prefer to have the tests in a test module

Reply:
Done